### DISCUSSION TITLE
想咨询下Alist画廊模式如何打开

### DISCUSSION THREAD

**Discussion by Meeeeet6** [AlistGo/alist#1058] [Ideas]
在Readme看到Alist支持画廊模式，想问问如何使用？
也有个小Idea，Alist能不能加入一个像手机相册（画廊）一样展示图片的，希望PC和手机页面都能支持，全显示缩略图那种，希望能支持一下~ 麻烦作者大大了，辛苦啦~

**Comment by Meeeeet6** [comment]
![image](https://user-images.githubusercontent.com/74483113/167283457-37552a7d-4bed-41c3-b507-f596cd68bb05.png)
就像这样子......

**Comment by xhofe** [comment]
![image](https://user-images.githubusercontent.com/36558727/167291757-ba87fd94-71a1-454a-9806-9ec0ae690315.png)

**Comment by ksws0134422** [comment]
本地硬盘并不会显示缩略图，云盘就可以

**Comment by anwen-anyi** [comment]
> 本地硬盘并不会显示缩略图，云盘就可以

不同的方式不一样 比如阿里云盘就会显示  其他别的就不会显示 是网盘的问题不是本程序控制的

**Comment by RikkaBlue** [comment]
为什么本地网盘无法显示预览呢。
